cmd/spec/main.go

```diff
@@ -46,6 +46,11 @@ func main() {
 		if name == "deletedAt" {
 			schema.Type = "string"
 			schema.Format = "date-time"
+			schema.OneOf = append(schema.OneOf, &openapi3.SchemaRef{Value: &openapi3.Schema{
```

Comment:
YOu have to do the same for the quickstart entity.

Reply:
ok, fixed

Reply:
I have also re-generated the specs